scipy/optimize/_differentialevolution.py

```diff
@@ -441,11 +452,25 @@ def x(self):
 
         Returns
         -------
-        x - ndarray
+        x : ndarray
             The best solution from the solver.
         """
         return self._scale_parameters(self.population[0])
 
+    @property
+    def convergence(self):
+        """
+        The convergence criterion.
+
+        Returns
+        -------
+        convergence : float
+            The standard deviation of the population energies divided by their
+            mean.
+        """
```

Comment:
Not sure what's the policy for documenting properties, but from a user POV, this would look strange to see a `Returns` section for something that is not callable. Maybe just keep the last sentence as a one-line docstring?